state/machine_ports.go

```diff
@@ -255,11 +255,19 @@ func (m *Model) OpenedPortRangesForAllMachines() ([]MachinePortRanges, error) {
 // getOpenedPortRangesForAllMachines returns a slice of machine port ranges for
 // all machines managed by this model.
 func getOpenedPortRangesForAllMachines(st *State) ([]*machinePortRanges, error) {
+	machines, err := st.AllMachines()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	var machineIDs []string
+	for _, m := range machines {
+		machineIDs = append(machineIDs, m.Id())
+	}
```

Comment:
Is this to filter out any caas port ranges?

Reply:
Yeah, here we only care about the machine port ranges.